docs/scenarios_authoring/DataSourcesAndSinks.md

```diff
@@ -34,8 +34,23 @@ The `kafka` sink configuration form will show a list of fields defined in Schema
 
 ![Kafka sink](img/kafkaSink.png "Kafka sink")
 
-If you prefer to define the whole output event as one expression containing value to be written to the Kafka topic, instead of filling separete autogenerated fields, you can do that by switching `Raw editor` to `true`.
+If you prefer to define the whole output event as one expression containing value to be written to the Kafka topic, instead of filling separate autogenerated fields, you can do that by switching `Raw editor` to `true`.
 
 When using the Raw editor, you can choose the validation mode:
-* strict - requires providing all fields, including optional fields, without redundant fields;
-* lax - requires providing only required fields, you can skip optional fields and add redundant fields.
\ No newline at end of file
+* strict - requires providing all fields, including optional fields, without redundant fields, requires exact matching;
+* lax - requires providing only required fields, you can skip optional fields and add redundant fields, exact matching is not required;
```

Comment:
I would prefer _omit_ instead of _skip_.   For me conveys the required meaning better - although this may be very subjective. Ask for a 3rd opinion, if not convinced.

redundant --> additional (if you meant [this](https://json-schema.org/understanding-json-schema/reference/object.html#additional-properties).

This _exact matching_ sounds mysterious. If difficult to define, may be the opposite is easier to define ("what is non exact matching")